libs/libmbim/Makefile

```diff
@@ -11,24 +11,28 @@ PKG_NAME:=libmbim
 PKG_VERSION:=1.26.2
 PKG_RELEASE:=$(AUTORELEASE)
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=https://www.freedesktop.org/software/libmbim
-PKG_HASH:=10c77bf5b5eb8c92ba80e9b519923ad9b898362bc8e1928e2bc9a17eeba649af
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.freedesktop.org/mobile-broadband/libmbim.git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=47eb6511818271be66336fb67ec97de7925119b09ddbd0ffd06b3f76893a461e
 
 PKG_MAINTAINER:=Nicholas Smith <nicholas@nbembedded.com>
 
 PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=python3/host
```

Comment:
Why?

Reply:
Meson build requires python3 https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/blob/main/meson.build#L48
Is this dependency not needed in this case?

Reply:
I don't think it's needed here. Try removing and see if it builds.

Reply:
@Linaro1985 could you open a new PR to remove that dep?

Reply:
Ah! great, thanks